cmds/truncate/truncate.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Truncate - shrink or extend the size of a file to the specified size
+//
+// Synopsis:
+//     truncate [OPTIONS] [FILE]...
+//
+// Options:
+//     -s: size in bytes
+//     -c: do not create any files
+//
+// Author:
+//     Roland Kammerer <dev.rck@gmail.com>
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+)
+
+var (
+	size     = flag.Int64("s", -1, "size in bytes")
+	nocreate = flag.Bool("c", false, "do not create files")
+)
+
+func main() {
+	flag.Parse()
+
+	if *size == -1 {
+		log.Fatalln("truncate: ERROR: You have to specify a non-negative size")
+	}
+
+	if flag.NArg() == 0 {
```

Comment:
You might want to say something like
"you need to specify one or more files"
as it's not clear from this message that it takes multiple files.